### PR TITLE
update visuals and dependencies

### PR DESCRIPTION
**Pull request by FredLL-Avaiga**
resolves #145

### CODE REVIEW

File: assets/TaipyLogoBW.svg
Comment (FabienLelaquais) [inline assets/TaipyLogoBW.svg:1]: I'm surprised that this image is not a square.
That might result in a skewed image.

File: l10n/bundle.l10n.json
Comment (FabienLelaquais) [inline l10n/bundle.l10n.json:1]: No newline at the end (?)

File: webviews/src/components/Editor.tsx [hunk 1]
```diff
@@ -14,7 +14,7 @@
 import { ChangeEvent, MouseEvent, useCallback, useEffect, useRef, useState } from "react";
 import { CanvasWidget } from "@projectstorm/react-canvas-core";
 import DomToImage from "dom-to-image-more";
-import * as deepEqual from "fast-deep-equal";
+import deepEqual from "fast-deep-equal";
 import * as l10n from "@vscode/l10n";
```
Comment (FabienLelaquais) [inline webviews/src/components/Editor.tsx:18]: '*', really?

**Review by FabienLelaquais — APPROVED** [review]
If the image is not skewed, we're good!
Well done

File: assets/TaipyLogoBW.svg
Comment (FredLL-Avaiga) [reply assets/TaipyLogoBW.svg:1]: image

File: l10n/bundle.l10n.json
Comment (FredLL-Avaiga) [reply l10n/bundle.l10n.json:1]: generated file

File: webviews/src/components/Editor.tsx (hunk 1 above)
Comment (FredLL-Avaiga) [reply webviews/src/components/Editor.tsx:18]: no choice here

File: assets/TaipyLogoBW.svg
Comment (FredLL-Avaiga) [reply assets/TaipyLogoBW.svg:1]: @FabienLelaquais is it good enough ?

**Review by FabienLelaquais — APPROVED** [review]

**Pull request merged**